### PR TITLE
Updated editorconfig file

### PR DESCRIPTION
**Pull request by gmponos**

### CODE REVIEW

**Comment by sagikazarmark** [comment]
Personally I prefer having a default rule: https://github.com/emperror/errors/blob/master/.editorconfig#L3-L9

**Comment by gmponos** [comment]
I prefer it too.. but it was PHP before.. changed now :)

File: .editorconfig [hunk 1]
```diff
@@ -5,7 +5,16 @@ root = true
...
+[*]
+charset = utf-8
 end_of_line = lf
 indent_style = space
 indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (Nyholm) [inline .editorconfig:17]: Why do we need this?

Comment (gmponos) [reply .editorconfig]: So when you first commented I thought "why not to trim on `md` files".. but then I saw that this was false. My bad.. so I updated this..

**Pull request merged**